L1Trigger/TrackFindingTracklet/src/MatchEngineUnit.cc

```diff
@@ -69,19 +140,28 @@ void MatchEngineUnit::step() {
     }
   }
 
+  // Detailed printout for comparison with HLS code
```

Comment:
If no need, please clean up. GIT will save history anyways. If it is still needed, maybe define DEBUG and use it to trigger the log.

Reply:
changed to messagelogger